angular.json

```diff
@@ -31,10 +31,25 @@
               "src/assets",
               "src/manifest.webmanifest"
             ],
-            "styles": ["src/styles.scss"],
+            "styles": [
+              "src/styles.scss",
+              {
+                "input": "src/theme/themes/orange/style.scss",
+                "lazy": true,
+                "bundleName": "orange"
+              },
+              {
+                "input": "src/theme/themes/blue/style.scss",
+                "lazy": true,
+                "bundleName": "blue"
+              }
+            ],
             "scripts": []
           },
           "configurations": {
+            "mock": {
+              "extractCss": true
```

Comment:
why not add this directly in the builder options above? That way no additional configuration would be required